inc/cleantalk-integrations-by-hook.php

```diff
@@ -424,10 +424,36 @@
         'hook'    => 'booked_add_appt',
         'setting' => 'forms__contact_forms_test',
         'ajax'    => true
-    ),
+    )
+);
+
+$apbct_active_rest_integrations = array(
+    'SureForms'         => array(
+        'rest_route'    => '/sureforms/v1/submit-form',
+        'setting' => 'forms__contact_forms_test',
+        'rest'       => true,
+    )
 );
 
-add_action('plugins_loaded', function () use ($apbct_active_integrations) {
+add_filter('rest_pre_dispatch', function ($result, $_, $request) use ($apbct_active_integrations, $apbct_active_rest_integrations) {
+    $route = $request->get_route();
+    foreach ($apbct_active_rest_integrations as $integration_name => $rest_data) {
+        if (isset($rest_data['rest_route']) && $rest_data['rest_route'] === $route) {
+            $all_integrations = array_merge($apbct_active_integrations, $apbct_active_rest_integrations);
```

Comment:
Why is it need to process all integrations instead of only rest-based integrations?